pkg/opensearch/client/search_request.go

```diff
@@ -422,7 +433,7 @@ const termsOrderTerm = "_term"
 func (b *aggBuilderImpl) Terms(key, field string, fn func(a *TermsAggregation, b AggBuilder)) AggBuilder {
 	innerAgg := &TermsAggregation{
 		Field: field,
-		Order: make(map[string]interface{}),
+		Size:  500,
```

Comment:
should the size be configurable? Maybe we can make a ticket for that later... also is there any risk this could like cause a breaking change for existing Term searches? Like If I wanted to group by a term and get the top 10 results, this wouldn't somehow cause me to get 500 right?

Reply:
You're totally right, I moved the size into the fn() where Terms aggregation is added